src/ReactiveUI/CommandBinding.cs

```diff
@@ -30,7 +33,7 @@ static CommandBinder()
         /// <param name="view">The View</param>
         /// <param name="viewModel">The View model</param>
         /// <param name="controlName">The name of the control on the view</param>
```

Comment:
There's inconsistency on what "controlName" is called, e.g. here we have {{controlName}}  but on line 189 we have a {{controlProperty}} - I think it would be worth being consistent...